improver/blending/utilities.py

```diff
@@ -62,3 +78,153 @@ def find_blend_dim_coord(cube, blend_coord):
         raise ValueError(msg)
 
     return cube.coord(dimensions=blend_dim[0], dim_coords=True).name()
+
+
+def get_coords_to_remove(cube, blend_coord):
+    """
+    Generate a list of coordinate names associated with the blend
+    dimension.  Unless these are time-related coordinates, they should be
+    removed after blending.
+
+    Args:
+        cube (iris.cube.Cube):
+            Cube to be blended
+        blend_coord (str):
+            Name of coordinate over which the blend will be performed
+
+    Returns:
+        list:
+            List of names of coordinates to remove
+    """
+    try:
+        (blend_dim,) = cube.coord_dims(blend_coord)
+    except ValueError:
+        # occurs if the blend coordinate is scalar
+        if blend_coord == MODEL_BLEND_COORD:
+            return [copy(MODEL_BLEND_COORD), copy(MODEL_NAME_COORD)]
```

Comment:
Similar to before, do these need to be copy statements?

Reply:
Possibly not - but I am a bit nervous about this as if the contents of the list are modified, it goes back and modifies the global variable.  I'm not confident around memory assignment in python: there seems to be a balance between reducing duplication by having a single central definition, but then that being vulnerable to overwriting by implication given python effectively assigns a pointer to the original string unless you copy.

I guess I could remove the copy here and see if the tests still pass?